Dialog docs (#2409)
microsoft/teams-ai: book/src/5.in-depth-guides/2.dialogs/3.handling-multi-step-forms.md

# Handling multi-step forms

> Dialogs can become complex and really powerful as they allow you to create multi-step forms. These forms can alter the flow of the survey depending on the user's input, or customize subsequent steps based on previous answers.

Reviewer: ```suggestion
Dialogs can become complex yet powerful with multi-step forms. These forms can alter the flow of the survey depending on the user's input or customize subsequent steps based on previous answers.
```
